thundra/plugins/trace/trace_plugin.py

```diff
@@ -182,7 +182,17 @@ def process_api_gw_response(self, plugin_context):
                 if not response.get('headers'):
                     response['headers'] = {}
 
-                response['headers'][constants.TRIGGER_RESOURCE_NAME_TAG] = plugin_context['request']['resource']
+                resource_path = None
```

Comment:
Can we use a utility method for extracting path from API GW request. In here,  `plugin_context['request']` will be passed and in the above (`inject_trigger_tags_for_api_gateway_proxy`) `original_event` will be passed